life_expectancy/tests/test_cleaning.py

```diff
@@ -1,16 +1,26 @@
 """Tests for the cleaning module"""
+
 import pandas as pd
 
-from life_expectancy.cleaning import main
-from . import OUTPUT_DIR
+from life_expectancy.cleaning import clean_data
 
 
-def test_clean_data(pt_life_expectancy_expected):
-    """Run the `clean_data` function and compare the output to the expected output"""
-    main()
-    pt_life_expectancy_actual = pd.read_csv(
-        OUTPUT_DIR / "pt_life_expectancy.csv"
-    )
+def test_cleaning(pt_life_expectancy_expected, eu_life_expectancy_raw_expected):
+    """Compare the output of the "clean_data" function to the expected output
+        Args:
+            pt_life_expectancy_expected (Fixture): load the expected output of the cleaning script
+            eu_life_expectancy_raw_expected (Fixture): load the expected raw data
+        Returns:
+    """
+    # Call the clean_data function and get the result
+    clean_data_actual = clean_data(
```

Comment:
I think that are better naming options to properly describe this variable. You could name it clean_data_output or clean_data_for_testing. These are just suggestions, I'm not the best at naming variables :-)

Reply:
Thanks! I have changed to clean_data_output_actual and  clean_data_output_expected :)